app/models/rating.rb

```diff
@@ -1,8 +1,19 @@
 # frozen_string_literal: true
 
 class Rating < ApplicationRecord
-  validates :rating, numericality: {only_integer: true, less_than_or_equal_to: 5, greater_than: 0}
+  SUBCATEGORIES = %i[originality description_quality test_quality model_solution_quality].freeze
+  CATEGORIES = SUBCATEGORIES + %i[overall_rating]
+
+  validates :overall_rating, numericality: {only_integer: true, less_than_or_equal_to: 5, greater_than: 0}
+  validates :originality, numericality: {only_integer: true, less_than_or_equal_to: 5, greater_than: 0}
+  validates :description_quality, numericality: {only_integer: true, less_than_or_equal_to: 5, greater_than: 0}
+  validates :test_quality, numericality: {only_integer: true, less_than_or_equal_to: 5, greater_than: 0}
+  validates :model_solution_quality, numericality: {only_integer: true, less_than_or_equal_to: 5, greater_than: 0}
 
   belongs_to :task
   belongs_to :user
+
+  def to_h
```

Comment:
Where are you using this method? While I like the method and simplicity, I am not finding any usage.

Depending on how it is used, I also wondered about potential alternatives, such as using `.attributes` rather than this `.to_h`.

Reply:
Yes, that is not needed anymore. I removed it :+1: